tf-psa-crypto/include/psa/crypto_config.h

```diff
@@ -147,4 +147,2595 @@
 #define PSA_WANT_KEY_TYPE_DH_KEY_PAIR_GENERATE  1
 //#define PSA_WANT_KEY_TYPE_DH_KEY_PAIR_DERIVE    1 /* Not supported */
 
```

Comment:
Please add the section tags for the `PSA_WANT` macros.

```suggestion
/** \} name SECTION Cryptographic mechanism selection (PSA API) */

```

for Line 32:
```
/**
 * \name SECTION Cryptographic mechanism selection (PSA API)
 * 
 * \{
 */
```